### DISCUSSION TITLE
[spacectl] Change user timezone by CLI

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#100] [Q&A] [answered]
If you are an administrator, then you want to change timezone of user. (This is not supported by UI).

CLI can change

~~~
spacectl exec update identity.User -p domain_id=dom-xxxxxxxx -p timezone="Asia/Seoul"
~~~

**Comment by whdalsrnt** [comment, marked as answer]
Hi, @choonho 

Currently, the user's timezone and language settings cannot be changed on the console's` Administrator > User` page.
This can only be changed on the user's `Profile` page.

However, you can change another user's timezone setting with the CLI, and the command is as follows.
`spacectl exec update identity.User -p domain_id=<user_domain_id> -p user_id=<user_id> -p timezone=<timezone>`

The above command is missing `user_id`.

Additionally, default timezone and language setting features for each domain are being developed and will be released in `1.11.0` version.
- Related Issue: https://github.com/orgs/cloudforet-io/projects/1/views/2?pane=issue&itemId=17814209
